### DISCUSSION TITLE
Matching labels and colors in Eclipse Tractus-X repositories

### DISCUSSION THREAD

**Discussion by FaGru3n** [eclipse-tractusx/sig-release#364] [Ideas]
As an Idea from the PCWM Workshop there was the idea to synchronize the colors and labels for each product repository.

In general a great idea i think, thanks to @jjeroch for mentioning.

In case we have default labels for our GitHub Organisation that could be verified in some newly generated repository like https://github.com/eclipse-tractusx/online-simulation-kit/labels

The Idea is to have the same issue labels and colors for the same labels in each repo.
![image](https://github.com/eclipse-tractusx/sig-release/assets/121097161/eff953d6-caa4-4eb3-aa87-ac7e731320af)

I guess we can either talk to Eclipse Foundation to synchronize or change the default labels.

But with the exiting repos we need a clear description which label should have which names and colors like:

| label name | color (HTML Code) | description             |
|------------|-------------------|-------------------------| 
| bug        | #d73a4a           | Something isn't working |

**Comment by evegufy** [comment]
I like the idea, aligned labels would support the overall overview und transparency.
I'd just like to keep the global administrative effort low at the same time, maybe this could be managed by using the [.eclipsefdn with otterdog](https://github.com/eclipse-tractusx/.eclipsefdn)?

**Reply by FaGru3n** [reply]
Add a issue against https://github.com/eclipse-tractusx/.eclipsefdn/issues/42 Otterdog, but did not find any key or value that matches against organisation labels
